### PR TITLE
#4 created skeleton project

### PR DESCRIPTION
**Pull request by ohdoking**

### CODE REVIEW

File: idiotquant/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .core import hmm
```
Comment (MinSikSon) [inline idiotquant/__init__.py:1]: hmm?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```
Comment (MinSikSon) [inline setup.py:1]: 이건 뭐하는 파일인겨?

Comment (MinSikSon) [inline setup.py:1]: 오씨 이거 설명좀 해줘

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+init:
+	pip install -r requirements.txt
```
Comment (MinSikSon) [inline Makefile:2]: requirements.txt 는 뭔디요?

File: idiotquant/__init__.py (hunk 1 above)
Comment (ohdoking) [reply idiotquant/__init__.py:1]: this is other package method name in core py. check this out

File: setup.py (hunk 2 above)
Comment (ohdoking) [reply setup.py:1]: Package and distribution management.

File: Makefile (hunk 3 above)
Comment (ohdoking) [reply Makefile:2]: that file is for external library

**Pull request merged**